marimo/_output/formatters/pandas_formatters.py

```diff
@@ -15,20 +15,28 @@ def package_name() -> str:
     def register(self) -> None:
         import pandas as pd
 
+        pd.set_option("display.max_rows", 10)
+        pd.set_option("display.max_columns", 20)
```

Comment:
this is just the new default right? and it is overridable by a user?

Reply:
Yes it's overridable, just the new default.